osquery/dispatcher/tests/scheduler.cpp

```diff
@@ -205,7 +205,7 @@ TEST_F(SchedulerTests, test_scheduler_zero_drift) {
 
   // Run the scheduler for 1 second with a second interval.
```

Comment:
If you update the comment below, maybe update this one to mention the drift arg as well to differentiate from the run above it?